scripts/new-msix-dev-cert.ps1

```
@@ -0,0 +1,78 @@
+
+.PARAMETER Password
+  PFX password (default: dev).
+
+.EXAMPLE
+  .\scripts\new-msix-dev-cert.ps1 -UpdateManifests
+  .\scripts\publish-msix.ps1 -CertificatePath "Desktop\HermesDesktop\packaging\dev-msix.pfx" -CertificatePassword dev
+#>
+param(
+    [switch] $UpdateManifests,
+
+    [string] $Password = "dev"
```

PSScriptAnalyzer (1.25.0)

[warning] Parameter '$Password' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $Password type to SecureString; Set $Password type to PSCredential

(PSAvoidUsingPlainTextForPassword)